packages/graph/hash_graph/lib/graph/src/datastore/postgres/mod.rs

```diff
@@ -0,0 +1,95 @@
+use sqlx::{PgPool};
+use error_stack::{IntoReport, Result};
+use crate::datastore::{Datastore, DatastoreError};
+use crate::types::{GraphElementIdentifier, GraphElementType};
+
+/// A Postgres-backed Datastore
+pub struct PostgresDatabase {
```

Comment:
We encountered this in hEngine a lot:
```rust
warning: item name starts with its containing module's name
  --> lib/graph/src/datastore/mod.rs:10:1
   |
10 | pub struct DatastoreError;
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#module_name_repetitions
```

I'm tempted to say we disable the lint so we avoid stuff like 
```rust
import postgres::Database as PostgresDatabase
```
Which we know screws up refactoring, intellisense, etc.

Reply:
I copied the wrong line 😄 , I'll update tomorrow. The lint I was talking about was the one that complains about structs having prefixes that are the same as the module

Reply:
I think it's called `module_name_repetitions`, let's disable it globally.